doc/lightning-listchannels.7.txt

```diff
@@ -28,29 +28,39 @@ On success, an object with a "channels" key is returned containing a list of 0
 or more objects.
 
 Each object in the list contains the following data:
-- 'source' - The node that initiated the channel creation
-- 'destination' - The node that was the target of the channel creation
-- 'short_channel_id' - The channel identifier
-- 'public' - Boolean value, is publicly available
-- 'satoshis' - 
-- 'message_flags' - 
-- 'channel_flags' - 
-- 'active' - 					
-- 'last_update' -
-- 'base_fee_millisatoshi' -
-- 'fee_per_millionth' -
-- 'delay' -
-
-If 'shcort_channel_id' is supplied and no matching channels are found, a
+
+- 'source' : The node that initiated the channel creation
+- 'destination' : The node that was the target of the channel creation
+- 'short_channel_id' : The channel identifier
+- 'public' : Boolean value, is publicly available
+- 'satoshis' : The amount of funds available in the channel
+- 'message_flags' : Bitfield showing the presence of optional fields in the
+'channel_update' message (BOLT #7)
+- 'channel_flags' : Bitfield showing the end of the channel the node is on
```

Comment:
Well, that's exactly the direction bit and the disable bit. Would it maybe be better to be less verbose and just referring them to BOLT 07?